crpc/js-node/rpc_grpc_pb.js

```diff
@@ -1,398 +0,0 @@
-// GENERATED CODE -- DO NOT EDIT!
```

Comment:
merge with previous commit

Reply:
But it's better to keep removes or moves in standalone commits.